content/en/blog/_posts/2024-04-29-structured-authentication-configuration-beta.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: blog
+title: "Kubernetes 1.30: Structured Authentication Configuration Moves to Beta"
+date: 2024-04-29
+slug: structured-authentication-moves-to-beta
+---
+
+**Author:** [Anish Ramasekar](https://github.com/aramase) (Microsoft)
+
+With Kubernetes 1.30, we (SIG Auth) are moving Structured Authentication Configuration to beta.
+
```

Comment:
Here's what I'd like to write:
```markdown
Today's article is about _authentication_: finding out who's performing a task, and checking
that they are who they say they are. Check back in tomorrow to find about what's new in
Kubernetes v1.30 around _authorization_ (deciding what someone can and can't access).
```
but that won't work, because the authz article is scheduled for the 25th. It would IMO really help tell a story to readers if the authn article comes out exactly one calendar day before the authz article.

This seems misaligned. I'll ask release comms to reconsider the dates.

Reply:
@sftim I've updated the date to `2024-04-25` and added the introduction you suggested.

Reply:
Thanks